src/main/java/com/blackduck/integration/detect/ApplicationUpdater.java

```diff
@@ -180,7 +180,8 @@ protected boolean selfUpdate() {
                             | InstantiationException 
                             | NoSuchMethodException 
                             | InvocationTargetException
-                            | IOException ex) {
+                            | IOException
+                            | NullPointerException ex) {
```

Comment:
@gopannabd Since we want to prevent the NPE from being thrown, I think we can omit any changes related to catching it here. Do you concur?

Reply:
"I concur!"